app.js

```diff
@@ -249,44 +277,292 @@ const app = Vue.createApp({
             return eq1 + eq2 + eq3 + eq4 + eq5 +eq6
         },
         baseScore() {
+            //define lookup table
```

Comment:
Can you please link the new specification / computational requirements? I have actually never seen proper requirements, only got some description sent by email. Do we already have proper requirements we can actually review the logic against?